bundle/compliance/cis_gcp/rules/cis_4_6/rule.rego

```diff
@@ -0,0 +1,23 @@
+package compliance.cis_gcp.rules.cis_4_6
+
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+
+default is_ip_forwarding_enabled = false
+
+# Ensure That IP Forwarding Is Not Enabled on Instances
+finding = result {
+	# filter
+	data_adapter.is_compute_instance
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(assert.is_false(is_ip_forwarding_enabled)),
```

Comment:
```
 **Exception:**
 Instances created by GKE should be excluded because they need to have IP forwarding enabled and cannot be changed.
 Instances created by GKE have names that start with "gke-".
 ```